src/main/java/uk/gov/nationalarchives/pdi/step/jena/ActionIfNoSuchField.java

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
I get warning about this being a Javadoc comment which is not valid here. It should be a plain comment like `/* .. */`. Javadoc comments should only precede "a class, field, constructor or method declaration" - see https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format

Reply:
Huh! Good spot. These are actually generated by the `license-maven-plugin`. Updating the plugin seems to fix the issue, I will send a commit for that too.